minima.gemspec

```diff
@@ -19,6 +19,22 @@ Gem::Specification.new do |spec|
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
 
+  spec.post_install_message = <<-msg
+
+----------------------------------------------
+Thank you for installing minima 2.0!
+
+Minima 2.0 comes with a breaking change that
+renders '<your-site>/css/main.scss' redundant.
+That file is now bundled with this gem as
+'<minima>/assets/css/main.scss'.
+
+More Information:
+https://github.com/jekyll/minima/issues/43
```

Comment:
This is great! Should we add something to the README telling folks how they can use their custom CSS instead, namely by moving their `css/main.scss` to `assets/css/main.scss` in their site source? Would be good I think.